### PR TITLE
feat-9 added weather card 

### PR DESCRIPTION
**Pull request by eidan66**

### CODE REVIEW

File: src/components/TransparentBox/style.jsx [hunk 1]
```diff
@@ -3,8 +3,8 @@ import colorConvert from 'color-convert';
 
 export const TransparentBox = styled.div`
   position: relative;
-  width: 50%;
-  height: 50%;
+  display: table;
+  width: ${({ width }) => (width ? width : 'auto')};
```
Comment (almogbhl) [inline src/components/TransparentBox/style.jsx:7]: instead auto => 100%;

File: src/components/TransparentBox/style.jsx [hunk 2]
```diff
@@ -3,8 +3,8 @@ import colorConvert from 'color-convert';
 
 export const TransparentBox = styled.div`
   position: relative;
-  width: 50%;
-  height: 50%;
+  display: table;
```
Comment (almogbhl) [inline src/components/TransparentBox/style.jsx:6]: why not flexbox? u need to start get used to it

File: src/components/WeatherCard/WeatherCard.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as S from './style';
+
+/* Import Components */
```
Comment (almogbhl) [inline src/components/WeatherCard/WeatherCard.jsx:4]: no need to comment this

File: src/components/WeatherCard/WeatherCard.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+import TransparentBox from '../TransparentBox';
```
Comment (almogbhl) [inline src/components/WeatherCard/WeatherCard.jsx:5]: just import everthing from './components

File: src/components/WeatherCard/WeatherCard.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+import Subtitle from '../Subtitle';
+import Icon from '../Icon';
+
+const WeatherCard = () => {
+  return (
+    <S.Card>
+      <TransparentBox color='lightgray' opacity='0.2'>
+        <Subtitle text='text1' />
```
Comment (almogbhl) [inline src/components/WeatherCard/WeatherCard.jsx:13]: maybe typography is a better name

File: src/components/WeatherCard/WeatherCard.jsx [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+
+/* Import Components */
+import TransparentBox from '../TransparentBox';
+import Subtitle from '../Subtitle';
+import Icon from '../Icon';
+
+const WeatherCard = () => {
+  return (
+    <S.Card>
+      <TransparentBox color='lightgray' opacity='0.2'>
```
Comment (almogbhl) [inline src/components/WeatherCard/WeatherCard.jsx:12]: maybe u mean backDrop?
https://css-tricks.com/almanac/properties/b/backdrop-filter/

File: src/components/WeatherCard/WeatherCard.jsx [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+        <Subtitle text='text1' />
+        <Subtitle text='text2' />
+        <Icon src='https://apidev.accuweather.com/developers/Media/Default/WeatherIcons/14-s.png' />
```
Comment (almogbhl) [inline src/components/WeatherCard/WeatherCard.jsx:15]: add an option for choosing an icon from the app assets

File: src/components/WeatherCard/style.jsx [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import styled from 'styled-components';
+
+export const Card = styled.div`
+  display: table-row;
```
Comment (almogbhl) [inline src/components/WeatherCard/style.jsx:4]: why not flexbox?

**Review by almogbhl — CHANGES_REQUESTED** [review]

File: src/components/WeatherCard/WeatherCard.jsx (hunk 4 above)
Comment (eidan66) [reply src/components/WeatherCard/WeatherCard.jsx:5]: I need create new file (`index.jsx`) in components folder and export all components from there ?

File: src/components/WeatherCard/WeatherCard.jsx (hunk 5 above)
Comment (eidan66) [reply src/components/WeatherCard/WeatherCard.jsx:13]: I have 2 typography components: 
1- Title
2- Subtitle

Can i move them to folder with typography name and from them export them with `index,jsx` file?

File: src/components/WeatherCard/WeatherCard.jsx (hunk 7 above)
Comment (eidan66) [reply src/components/WeatherCard/WeatherCard.jsx:15]: i fix it here : #14

**Pull request merged**